components/bank/components/tokenList.tsx

```
@@ -43,10 +43,19 @@ export function TokenList(props: Readonly<TokenListProps>) {
     chains,
   } = props;
   const [selectedDenom, setSelectedDenom] = useState<any>(null);
-  const [isSendModalOpen, setIsSendModalOpen] = useState(false);
+  const [isSendModalOpen, setIsSendModalOpenHook] = useState(false);
   const [currentPage, setCurrentPage] = useState(1);
   const [openDenomInfoModal, setOpenDenomInfoModal] = useState(false);
 
+  function setIsSendModalOpen(isOpen: boolean) {
+    if (isOpen === false) {
+      debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)